addons/binding/org.openhab.binding.nest/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,44 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.binding.nest.internal
+Ignore-Package: org.openhab.binding.nest.internal
+Bundle-ManifestVersion: 2
+Bundle-Name: Nest Binding
+Bundle-SymbolicName: org.openhab.binding.nest;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .
+Bundle-DocURL: http://www.openhab.org
+Import-Package: com.google.common.collect,
+ com.google.gson,
+ com.google.gson.annotations,
+ org.apache.commons.httpclient,
```

Comment:
As an HTTP client, the [Jetty lib](https://www.eclipse.org/smarthome/documentation/development/bindings/dependencies.html#for-http-operations) should actually be used in favor of the apache httpclient.